assets/js/base/context/cart-checkout/payment-methods/payment-method-data-context.js

```diff
@@ -91,13 +91,7 @@ const getCustomerPaymentMethods = ( availablePaymentMethods = {} ) => {
 	const enabledCustomerPaymentMethods = {};
 	paymentMethodKeys.forEach( ( type ) => {
 		const methods = customerPaymentMethods[ type ].filter(
-			( { method: { gateway } } ) => {
-				const isAvailable = gateway in availablePaymentMethods;
-				return (
-					isAvailable &&
-					availablePaymentMethods[ gateway ].supports?.savePaymentInfo
-				);
-			}
```

Comment:
It's still fuzzy to me what this check does exactly? and why did it had `supports?.savePaymentInfo`?